### PR TITLE
Add two new variables to the Helm chart

### PR DESCRIPTION
**Pull request by bodgit**
* Allow customising the `revisionHistoryLimit` field in the Deployment, defaulting to the Kubernetes default of 10.
* Allow customising the volume used for storing the default trust package, defaulting to the original `emptyDir: {}` value.

### CODE REVIEW

File: deploy/charts/trust-manager/templates/deployment.yaml [hunk 1]
```diff
@@ -120,7 +121,7 @@ spec:
       {{- end }}
       volumes:
       - name: packages
-        emptyDir: {}
+        {{- toYaml .Values.packagesVolume | nindent 8 }}
```
Comment (inteon) [inline deploy/charts/trust-manager/templates/deployment.yaml:124]: Could you give an example of how and why you would change this value?

**Comment by bodgit** [comment]
@inteon did you have any further feedback?

File: deploy/charts/trust-manager/templates/deployment.yaml (hunk 1 above)
Comment (SgtCoDFish) [reply deploy/charts/trust-manager/templates/deployment.yaml:129]: Hey, just seen this PR - I think we'll go in a different direction with this by setting a size limit unilaterally in #308  - appreciate your involvement. If you wanted to reword this to only focus on revisionHistoryLimit (along with maybe some justification for why we should change that) we'd be happy to take a look!

**Comment by SgtCoDFish** [comment]
Gonna close this since we went a different way with it! Thanks so much @bodgit for raising something - appreciate you getting involved even if we didn't merge this ❤️

**Comment by bodgit** [comment]
> Gonna close this since we went a different way with it! Thanks so much @bodgit for raising something - appreciate you getting involved even if we didn't merge this ❤️

I will redo the PR for just the `revisionHistoryLimit` settings.

**Comment by SgtCoDFish** [comment]
Ah sorry - I missed that part entirely!

Yeah if you raise a PR I'll take a look at that. Please feel free to assign it to me and ping me for a review!
